.circleci/config.yml

```diff
@@ -64,6 +64,7 @@ commands:
             - "packages/bot-skeleton/node_modules"
             - "packages/bot-web-ui/node_modules"
             - "packages/cashier/node_modules"
+            - "packages/reports/node_modules"
```

Comment:
Not that it is important but seems like the list's been in alphabetical order 😄

Reply:
right, will be fixed :pray: